docs/src/token-2022/presentation.md

```diff
@@ -25,7 +25,7 @@ title: Presentation
 
 ---
 
-### Are you aware that it's 2023?
+### Are you aware that it's 2024?
```

Comment:
Since it'll soon be 2025, how about we future-proof this :smile: 
```suggestion
### Are you aware that it's not 2022?
```

Reply:
Great idea, should've thought of this earlier 😅